packages/project-management-automation/lib/tasks/add-milestone/index.js

```diff
@@ -41,26 +38,19 @@ const isDuplicateValidationError = ( error ) =>
  * @param {string} repo    Repository name.
  * @param {string} title   Milestone title.
  *
- * @return {Promise<OktokitIssuesListMilestonesForRepoResponseItem|void>} Promise resolving to milestone, if exists.
+ * @return {Promise<any|void>} Promise resolving to milestone, if exists.
```

Comment:
Not sure how these types are defined now, but `import('@octokit/rest').IssuesListMilestonesForRepoResponseItem` no longer works.